modules/localgov_netcall_ai_widget/config/schema/localgov_netcall_ai_widget.schema.yml

```diff
@@ -0,0 +1,8 @@
+# Schema for the configuration files of the Content Access module.
+localgov_netcall_ai_widget.settings:
+  type: config_object
+  label: 'Netcall AI Widget settings'
+  mapping:
+    roles:
+      type: checkboxes
```

Comment:
As far as I remember, the `type` here refers to a data type.  [Potential values](https://www.drupal.org/docs/drupal-apis/configuration-api/configuration-schemametadata#types) include boolean, integer, etc.  Please try the [config_inspector](https://www.drupal.org/project/config_inspector) module to validate this.  If config_inspector spots an error, try the following snippet which is from the [views](https://git.drupalcode.org/project/drupal/-/blob/11.x/core/modules/views/config/schema/views.data_types.schema.yml?ref_type=heads#L782) module:
```
roles:
  type: sequence
  label: 'User roles'
  sequence:
    type: string
    label: 'Role'
```

Reply:
Thanks Adnan, a great spot from you. This should be fixed via a00b18a